x-pack/plugins/cases/common/openapi/entrypoint.yaml

```diff
@@ -0,0 +1,85 @@
+paths:
+  /api/cases:
+    $ref: paths/api@cases.yaml
+#  /api/cases/_find:
+#    $ref: paths/api@cases@_find.yaml
+#  '/api/cases/alerts/{alertId}':
+#    $ref: 'paths/api@cases@alerts@{alertid}.yaml'
+#  '/api/cases/configure':
+#    $ref: paths/api@cases@configure.yaml
+#  '/api/cases/configure/{configurationId}':
+#    $ref: paths/api@cases@configure@{configurationid}.yaml
+#  '/api/cases/configure/connectors/_find':
+#    $ref: paths/api@cases@configure@connectors@_find.yaml
+#  '/api/cases/reporters':
+#    $ref: 'paths/api@cases@reporters.yaml'
+#  '/api/cases/status':
+#    $ref: 'paths/api@cases@status.yaml'
+#  '/api/cases/tags':
+#    $ref: 'paths/api@cases@tags.yaml'
+#  '/api/cases/{caseId}':
+#    $ref: 'paths/api@cases@{caseid}.yaml'
+#  '/api/cases/{caseId}/alerts':
+#    $ref: 'paths/api@cases@{caseid}@alerts.yaml'
+#  '/api/cases/{caseId}/comments':
+#    $ref: 'paths/api@cases@{caseid}@comments.yaml'
+#  '/api/cases/{caseId}/comments/{commentId}':
+#    $ref: 'paths/api@cases@{caseid}@comments@{commentid}.yaml'
+#  '/api/cases/{caseId}/connector/{connectorId}/_push':
+#    $ref: 'paths/api@cases@{caseid}@connector@{connectorid}@_push.yaml'
+#  '/api/cases/{caseId}/user_actions':
+#    $ref: 'paths/api@cases@{caseid}@user_actions.yaml'
+  
+  '/s/{spaceId}/api/cases':
+    $ref: 'paths/s@{spaceid}@api@cases.yaml'
+ # '/s/{spaceId}/api/cases/_find':
+ #   $ref: 'paths/s@{spaceid}@api@cases@_find.yaml'
+ # '/s/{spaceId}/api/cases/alerts/{alertId}':
+ #   $ref: 'paths/s@{spaceid}@api@cases@alerts@{alertid}.yaml'
+ # '/s/{spaceId}/api/cases/configure':
+ #   $ref: paths/s@{spaceid}@api@cases@configure.yaml
+ # '/s/{spaceId}/api/cases/configure/{configurationId}':
+ #   $ref: paths/s@{spaceid}@api@cases@configure@{configurationid}.yaml
+ # '/s/{spaceId}/api/cases/configure/connectors/_find':
+ #   $ref: paths/s@{spaceid}@api@cases@configure@connectors@_find.yaml
+ # '/s/{spaceId}/api/cases/reporters':
+ #   $ref: 'paths/s@{spaceid}@api@cases@reporters.yaml'
+ # '/s/{spaceId}/api/cases/status':
+ #   $ref: 'paths/s@{spaceid}@api@cases@status.yaml'
+ # '/s/{spaceId}/api/cases/tags':
+ #   $ref: 'paths/s@{spaceid}@api@cases@tags.yaml'
+ # '/s/{spaceId}/api/cases/{caseId}':
+ #   $ref: 'paths/s@{spaceid}@api@cases@{caseid}.yaml'
+ # '/s/{spaceId}/api/cases/{caseId}/alerts':
+ #   $ref: 'paths/s@{spaceid}@api@cases@{caseid}@alerts.yaml'
+ # '/s/{spaceId}/api/cases/{caseId}/comments':
+ #   $ref: 'paths/s@{spaceid}@api@cases@{caseid}@comments.yaml'
+ # '/s/{spaceId}/api/cases/{caseId}/comments/{commentId}':
+ #   $ref: 'paths/s@{spaceid}@api@cases@{caseid}@comments@{commentid}.yaml'
+ # '/s/{spaceId}/api/cases/{caseId}/connector/{connectorId}/_push':
+ #   $ref: 'paths/s@{spaceid}@api@cases@{caseid}@connector@{connectorid}@_push.yaml'
+ # '/s/{spaceId}/api/cases/{caseId}/user_actions':
+ #   $ref: 'paths/s@{spaceid}@api@cases@{caseid}@user_actions.yaml'
+components:
+  securitySchemes:
+    basicAuth:
+      type: http
```

Comment:
This can also be HTTPS.

Reply:
That seems to just be one of four security schema keywords (along with "apiKey", "oauth2", and "openIdConnect"). The "http" type is for basic, bearer and other HTTP authentications schemes per https://swagger.io/docs/specification/authentication/.  I think per https://www.elastic.co/guide/en/kibana/master/api.html#key-authentication we likely need to add the "apiKey" type here too.

Reply:
Ok!